scripts/dev/dockerfiles/Dockerfile.readiness

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.13-alpine as builder
+
+COPY ./cmd/readiness /build/
+COPY ./pkg /build/pkg
+COPY ./api /build/api
+COPY ./go.mod /build/
```

Comment:
So we have different go.mod file for the probe, can/should we use the project's go.mod?

Reply:
this is a single go.mod file, we just have a different main (in this case `cmd/readiness/main.go`)